hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/operations/Backup.java

```diff
@@ -154,12 +158,12 @@ public void run() throws Exception {
 
     @Override
     public void afterRun() throws Exception {
-        if (!valid || !sync || getCallId() == 0 || originalCaller == null) {
+        if (!valid || !sync || backupOp.getCallId() == 0 || originalCaller == null) {
```

Comment:
What if `backupOp` is `null`? According to the `run()` method, that seems possible.

```java
if (backupOp == null && backupOpData != null) {
    backupOpUpdater.lazySet(this, (Operation) nodeEngine.getSerializationService().toObject(backupOpData));
}

// I THINK AT THIS POINT backupOp SHOULD NOT ALLOWED TO BE null.

if (backupOp != null) {
    ensureBackupOperationInitialized();

    backupOp.beforeRun();
    backupOp.run();
    backupOp.afterRun();
}
```